netrd/distance/quantum_jsd.py

```diff
@@ -20,14 +20,16 @@
 import numpy as np
 from scipy.linalg import expm
 from .base import BaseDistance
-from ..utilities import ensure_undirected, ensure_unweighted
+from ..utilities import undirected, unweighted
 
 
 class QuantumJSD(BaseDistance):
     """Compares the spectral entropies of the density matrices."""
 
+    @undirected
```

Comment:
Does `wraps` work well when chaining decorators like this?

Reply:
I think it does, but I'll admit I didn't check that. Will report back later.

Reply:
A lazy update on this: [this should be the purpose of using `wraps()`](https://stackoverflow.com/questions/739654/how-to-make-a-chain-of-function-decorators).